XYZ accelerometer
XYZ magnometer
XYZ gyrometer
